### PR TITLE
Feature update modulefiles

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
I tested loading these new modulefiles on the various machines.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Please review the files for any obvious mistakes.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[N/A]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Do these changes introduce new SonarQube findings? **[No]**</br>
If **yes**, please describe:

- [x] Please complete this pull request review by **[20240503]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [No issue] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [No issue] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: internal/scripts/installation/modulefiles/3.0.0_acorn
Comment (bikegeek) [inline internal/scripts/installation/modulefiles/3.0.0_acorn:1]: I think the prepend-path for PATH and PYTHONPATH needs an entry for METdbLoad

File: internal/scripts/installation/modulefiles/3.0.0_derecho
Comment (bikegeek) [inline internal/scripts/installation/modulefiles/3.0.0_derecho:1]: I think the prepend-path for PATH and PYTHONPATH needs an entry for METdbLoad

File: internal/scripts/installation/modulefiles/3.0.0_gaea
Comment (bikegeek) [inline internal/scripts/installation/modulefiles/3.0.0_gaea:1]: I think the prepend-path for PATH and PYTHONPATH needs an entry for METdbLoad

File: internal/scripts/installation/modulefiles/3.0.0_hera
Comment (bikegeek) [inline internal/scripts/installation/modulefiles/3.0.0_hera:1]: I think the prepend-path for PATH and PYTHONPATH needs an entry for METdbLoad

File: internal/scripts/installation/modulefiles/3.0.0_hercules
Comment (bikegeek) [inline internal/scripts/installation/modulefiles/3.0.0_hercules:1]: I think the prepend-path for PATH and PYTHONPATH needs an entry for METdbLoad

File: internal/scripts/installation/modulefiles/3.0.0_jet
Comment (bikegeek) [inline internal/scripts/installation/modulefiles/3.0.0_jet:1]: I think the prepend-path for PATH and PYTHONPATH needs an entry for METdbLoad

File: internal/scripts/installation/modulefiles/3.0.0_orion
Comment (bikegeek) [inline internal/scripts/installation/modulefiles/3.0.0_orion:1]: I think the prepend-path for PATH and PYTHONPATH needs an entry for METdbLoad

**Comment by jprestop** [comment]
Thank you, @bikegeek.  Is METdbLoad only for database functionality?  If so, I think it is true that METdbLoad would not be used on WCOSS2, in which case, I will add prepend-path for PATH and PYTHONPATH for all but the 3.0.0_acorn file (the WCOSS2 test machine).  If not, I will add it to all.  I will likely not get back to this until next week, but I will follow up and request a re-review then.

**Comment by bikegeek** [comment]
The METdbLoad has some functionality used by the METreformat code to read
in MET .stat and .tcst files.

**Comment by jprestop** [comment]
@bikegeek I have added entries to the prepend-path for PATH and PYTHONPATH to all of the files.  I have a requested a re-review.  There is no rush.  Thank you for your attention to detail with these modulefiles.

**Review by bikegeek — APPROVED** [review]
looks good, thanks !

**Pull request merged**